CadRevealComposer/ModelMetadata.cs

```diff
@@ -26,4 +27,23 @@ public static string Serialize(ModelMetadata metadata)
     {
         return JsonSerializer.Serialize(metadata._metadata, JsonSerializerOptions);
     }
+
+    // returns true if and only if the value in metadata equals the value passed in the arg
+    // returns false otherwise
+    public bool CheckValue(string key, string value)
+    {
+        string? realValue;
+        if (_metadata.TryGetValue(key, out realValue))
+        {
+            if (realValue!.Equals(value, System.StringComparison.OrdinalIgnoreCase))
```

Comment:
Instead of the if/else this could be just:
`return realValue!.Equals(value, System.StringComparison.OrdinalIgnoreCase);`